cifar10/utils.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import json
+import os
+
+from sklearn.metrics import confusion_matrix
+from io import StringIO
+import boto3
+
+
+class Visualization:
```

Comment:
This class is created as a utility to view visualization. The current code structure makes it impossible to keep the `utils.py` as a common file. Hence, placed it in cifar10/utils.py. Will work on the code revamp and then move this file to a common location.